protocol-specs/generators/block_processing_second_signature/index.js

```diff
@@ -200,22 +207,26 @@ const generateTestCasesValidBlockSecondSignatureTx = () => {
 
 const generateTestCasesinvalidBlockWithSecondSignatureAndFundsTxSuite = () => {
 	const amount = '5500000000';
-	const transferTx = new TransferTransaction(
-		transfer({
-			amount,
-			passphrase: accounts.genesis.passphrase,
-			recipientId: accounts.existingDelegate.address,
-		}),
-	);
+	const transferObject = transfer({
```

Comment:
same as below. Do we need to mix this function and the instance?
It was meant to be used one way or another

Reply:
It was faster to implement this way with the least amount of changes I will take the extra time for doing it from JSON/Object literal to instance.

Reply:
Fixed.